tests/fixtures/circulation.py

```diff
@@ -362,6 +362,50 @@ def patron2_martigny_no_email(
     return ptrn
 
 
+# ------------ Org: Martigny Patron 3 (blocked) ----------
+@pytest.fixture(scope="module")
+def patron3_martigny_data(data):
+    """Load Martigny patron data."""
```

Comment:
Here you are creating a conflict with `patron_2_martigny` because both of them share the same pid, barcode and other data. So no one will be able to use `patron_2_martigny` and `patron_3_martigny` in same test.
I suggest that you create a new `ptrn11` and assign it to `patron3_martigny`